.changes/v3.7.0/841-improvements.md

```diff
@@ -0,0 +1,7 @@
+* Add VDC Group compatibility for `resource/vcd_nsxt_firewall` and `datasource/vcd_nsxt_firewall` [GH-841]
```

Comment:
> Other things are just cosmetics/cleanup and consist of:
> 
>     Deprecating vdc field in all those resources and data sources (including removing fields from docs)

I actually find this not just a cosmetic change :) So, given that the users are quite accustomed to the `vdc` field, I suggest we add a big changelog entry at the top for this change with an explanation/justification of why it was done.

Reply:
In fact we have a forgotten `deprecations` section particularly for changelog. I have put additional details there - https://github.com/vmware/terraform-provider-vcd/pull/841/files#diff-03eefc5b761664186e166ce83780280786734ff0ae4174bb5655619670c49b0f

Reply:
Thanks! One comment:

> This field is no longer precise as with introduction of VDC Group support an Edge Gateway can be bound either to a VDC, either to a VDC Group

After reading this, it's still not clear from where these resources get VDC or VDC Group. What about adding a sentence that it's inherited from the parent EG?

Reply:
Added.